couchdb/src/main/java/org/hibernate/ogm/datastore/couchdb/CouchDBDialect.java

```diff
@@ -55,9 +55,9 @@
 /**
```

Comment:
Why is the `CouchDBDialect` modified at all? Is there any change besides formatting?